spec/pickup_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EasyPost::Pickup do
+  describe '.create' do
+    it 'creates a pickup' do
+      shipment = EasyPost::Shipment.create(Fixture.one_call_buy_shipment)
+
+      pickup_data = Fixture.basic_pickup
+      pickup_data[:shipment] = shipment
+
+      pickup = described_class.create(pickup_data)
+
+      expect(pickup).to be_an_instance_of(described_class)
+      expect(pickup.id).to match('pickup_')
+      expect(pickup.pickup_rates).not_to be_nil
+    end
+  end
+
+  describe '.retrieve' do
+    it 'retrieves a pickup' do
+      shipment = EasyPost::Shipment.create(Fixture.one_call_buy_shipment)
+
+      pickup_data = Fixture.basic_pickup
+      pickup_data[:shipment] = shipment
+
+      pickup = described_class.create(pickup_data)
+
+      retrieved_parcel = described_class.retrieve(pickup.id)
```

Comment:
I think the variable name should be **retrieved_pickup**.

Reply:
Yep, more copy/pasta